_extensions/preprint/typst-template.typ

```diff
@@ -260,5 +260,5 @@
 // Remove gridlines from tables
 #set table(
   inset: 6pt,
-  stroke: none
+  stroke: none,
```

Comment:
Thank you, can you get rid of the trailing comma and I'll accept.

Reply:
I will  fix this by the end of the day

Reply:
This is `typstyle` formatter issue, I created relevant issue https://github.com/Enter-tainer/typstyle/issues/147, meantime I can change formatter to `typstfmt`, what is the code formater you are using @mvuorre?

Reply:
@mvuorre apparently this is the correct syntax https://typst.app/docs/tutorial/advanced-styling/ (see also the comment in the issue I created on the formatter repo)

Reply:
The trailing comma is the standard practice adopted by the formatter developer.  and  often used in typst examples https://typst.app/docs/tutorial/advanced-styling/ 
@mvuorre  let me know how you want to proceed
1) accept `typstyle` formatted code with trailing coma without changes
2) accept ``typstyle` formatted code but with formatting turned off by `//@typstyle off` at specific line
3)  I can change formatter to `typstfmt` it will not add trailing comma but change the line to `#set table(inset: 6pt, stroke: none)`
4) other solution? 

I will make relevant  changes  if needed and add to this PR